device-types/Fortinet/FG-3700D.yaml

```diff
@@ -0,0 +1,97 @@
+---
+manufacturer: Fortinet
+model: FortiGate 3700D
+slug: fortinet-fg-3700d
+part_number: FG-3700D
+u_height: 3
+weight: 21.3
+weight_unit: kg
+airflow: front-to-rear
+is_full_depth: false
+front_image: true
+comments: '[FortiGate 3700D Series Datasheet](https://www.fortinet.com/content/dam/fortinet/assets/data-sheets/FortiGate_3700D.pdf)'
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: USB MGMT
+    type: usb-mini-b
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 840
+    allocated_draw: 300
+  - name: PS2
+    type: iec-60320-c14
+    maximum_draw: 840
+    allocated_draw: 300
```

Comment:
It appears that these PSUs are hot swappable. These should thus be modelled under `module-bays`, and then the associated PSU modules should be modelled in the module-types folder if they do not already exist there